main.go

```diff
@@ -241,7 +255,12 @@ func initServer(logger *zap.Logger, apmTracer *apm.Tracer, config *Config) *http
 	router := mustLoadRouter(logger, config, indexer)
 	apmgorilla.Instrument(router, apmgorilla.WithTracer(apmTracer))
 
-	return &http.Server{Addr: address, Handler: router}
+	if tlsMinVersion == "" {
+		return &http.Server{Addr: address, Handler: router}
+	}
+
+	TLSConfig := &tls.Config{MinVersion: tlsMinVersionCode}
+	return &http.Server{Addr: address, Handler: router, TLSConfig: TLSConfig}
```

Comment:
Nit. Special case inside the if.
```suggestion
	if tlsMinVersion == "" {
		TLSConfig := &tls.Config{MinVersion: tlsMinVersionCode}
		return &http.Server{Addr: address, Handler: router, TLSConfig: TLSConfig}
	}
	
	return &http.Server{Addr: address, Handler: router}
```
Or maybe better, use the if only to set the related setting, and keep a single return.
```suggestion
	var tlsConfig tls.Config
	if tlsMinVersion == "" {
		tlsConfig.MinVersion = tlsMinVersionCode
	}

	return &http.Server{Addr: address, Handler: router, TLSConfig: &tlsConfig}
```

Btw, it is a bit unexpected to check `tlsMinVersion` but later use `tlsMinVersionCode`.

Reply:
Used the second approach, thanks!